wiki/introduction/get-started/en.md

```diff
@@ -32,19 +42,23 @@ Asking questions properly, whether it is on our forum or on Discord, is an essen
 skill to have. Never be afraid to ask questions, we're always willing to help, and we
 don't judge.
 
-TODO: Methodology for asking questions here
+When asking questions, try to include the necessary context in your first message, and [do not ask to ask](https://dontasktoask.com/). State your problem, what you have already tried to solve it, and how those attempts failed. Make sure to avoid the [XY problem](https://xyproblem.info/) by not only asking for help for one specific part of a solution to your problem, but also stating the actual problem you are trying to solve. When debugging, include your `latest.log` and when your game crashes, a crash log. Additionally, full access to your source code can be really helpful, ideally uploaded to a version control system such as GitHub.
```

Comment:
```suggestion
When asking questions, try to include the necessary context in your first message, and [do not ask to ask](https://dontasktoask.com/). State your problem, what you have already tried to solve it, and how those attempts failed. Make sure to avoid the [XY problem](https://xyproblem.info/) by not only asking for help for one specific part of a solution to your problem, but also stating the actual problem you are trying to solve. When debugging, include your `latest.log` and when your game crashes, a crash log. Additionally, full access to your source code can be really helpful, ideally uploaded to a website such as GitHub.
```
the VCS is git, website is github